driver/reducecalculation_driver.hpp

```diff
@@ -151,10 +207,28 @@ int ReduceCalculationDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[]
 {
     inflags.Parse(argc, argv);
 
+    reduceCalculationOp =
+        static_cast<miopenReduceCalculationOp_t>(inflags.GetValueInt("ReduceCalculationOp"));
+
+    if(reduceCalculationOp < 1 || reduceCalculationOp > 4)
```

Comment:
`reduceCalculationOp` is type `miopenReduceCalculationOp_t`, not type `ReduceCalculationOp_t`. `miopenReduceCalculationOp_t` enum doesnt include `First_` (because I dont see previous enum structs do so). So I guess let's just keep it as it is for now?